src/components/Generic/Dialogs/AddOrUpdatePrinterDialog.vue

```diff
@@ -245,6 +246,22 @@ export default defineComponent({
     },
   },
   methods: {
+    onHostNamePaste(pasted: ClipboardEvent) {
+      try {
+        if (pasted.clipboardData != undefined) {
+          const text = pasted.clipboardData.getData("text");
+          const parsedUrl = new URL(text);
+          if (PrintersService.isValidPrinterUrl(parsedUrl)) {
+            /* Because @paste Event is handled after v-model data bind so the textfield gets updated two times first when
+             we update the v-model and then the textfield value adds the pasted value */
+            pasted.preventDefault();
+            PrintersService.applyPastedPrinterUrl(parsedUrl, this.formData);
+          }
+        }
+      } catch {
+        console.log("Invalid Printer URL");
```

Comment:
Such a log gets lost, especially with #197. Let's either:
- avoid the try-catch and fix the issue mentioned
- or show a validation error at the bottom of the form

Reply:
This log should be also removed. For the try catch i think it is nessecary as it catch if there is an error with the pated url as a whole. 
Pasting something like: 
Htgp://192.16+.8h.102:57-2 
It would catch wrong protocol, hostname and port

And the form already shows an error so i dont think it is nessecary

Reply:
In the meanwhile Im tackling #197 regardless. The app error handling needs a massive bump regardless.